paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentSheetScreen.kt

```diff
@@ -481,11 +482,14 @@ private fun PrimaryButton(viewModel: BaseSheetViewModel) {
             val binding = StripeFragmentPrimaryButtonContainerBinding.inflate(inflater, parent, attachToParent)
             val primaryButton = binding.primaryButton
             button = primaryButton
-            @Suppress("DEPRECATION")
             primaryButton.setAppearanceConfiguration(
                 StripeTheme.primaryButtonStyle,
-                tintList = viewModel.config.primaryButtonColor ?: ColorStateList.valueOf(
-                    StripeTheme.primaryButtonStyle.getBackgroundColor(context)
+                tintList = ColorStateList.valueOf(
```

Comment:
This change seems odd! Why do we need isSystemDarkTheme now, but we didn't before?